test_runner/fixtures/neon_fixtures.py

```diff
@@ -530,6 +530,16 @@ def available_remote_storages() -> List[RemoteStorageKind]:
     return remote_storages
 
 
+def available_s3_storages() -> List[RemoteStorageKind]:
```

Comment:
Can it be a parameter to available_remote_storages above?

Reply:
It can, but I decided that it is simpler to keep them as separate functions
and avoid changes in existing `available_remote_storages()` calls.

Reply:
You can avoid changes if parameter has a default value.